client/src/app/core/repositories/motions/motion-line-numbering.service.ts

```diff
@@ -485,6 +503,10 @@ export class MotionLineNumberingService {
 
         return baseParagraphs.map((origText: string, paraNo: number): string => {
             const newText = amendment.amendment_paragraph(paraNo);
+            /**
+             * TODO CHECK please check if this commit was coverd justly
+             * https://github.com/OpenSlides/OpenSlides/commit/ac50d6f8dccc5b502c7b7a3a106ef5935184494f#diff-f5bbc561818c722671cf3105c1c578d9e6b9767bb0f4f9ea32030678eb803499R1016-R1018
+             */
```

Comment:
This part seems not compatible with #101 
@GabrielInTheWorld please check
https://github.com/OpenSlides/OpenSlides/commit/ac50d6f8dccc5b502c7b7a3a106ef5935184494f#diff-f5bbc561818c722671cf3105c1c578d9e6b9767bb0f4f9ea32030678eb803499R1016-R1018

and make according changes